### PR TITLE
[Website] Add sizing to TFC partner badge

### PR DESCRIPTION
**Pull request by cgriggs01**
Adding sizing to the TFC integration badge for the partner program.

### CODE REVIEW

**Review by laurapacilio — APPROVED** [review]

File: content/docs/partnerships.mdx [hunk 1]
```diff
@@ -164,7 +164,7 @@ Partners who successfully complete the Terraform Cloud Integration Checklist obt
 
 - Note: Currently, pre-apply is the only integration phase available at this time. As of September 2021, run tasks are available only as a beta feature, are subject to change, and not all customers will see this functionality in their Terraform Cloud organization since this is currently enabled by default for our business tier customers of Terraform Cloud. If you have a customer that is interested in run tasks and are not a current Terraform Cloud for Business customer, customers can [sign up here](https://docs.google.com/forms/d/e/1FAIpQLSf3JJIkU05bKWov2wXa9c-QV524WNaHuGIk7xjHnwl5ceGw2A/viewform).
 
-![TFC Badge](/img/docs/tfc-badge.png)
+<img src="/img/docs/tfc-badge.png" style={{display:'block', margin:'auto', width:'200px', height:'200px'}}/>
```
Comment (dstaley) [inline content/docs/partnerships.mdx:167]: Can you please add the `alt` attribute? Thank you!

**Review by dstaley — CHANGES_REQUESTED** [review]

File: content/docs/partnerships.mdx (hunk 1 above)
Comment (laurapacilio) [reply content/docs/partnerships.mdx:167]: ahh - I'll remember to ask for this in future reviews - thank you Dylan!

Comment (cgriggs01) [reply content/docs/partnerships.mdx:167]: Thanks for the callout @dstaley, but what is the `alt`  attribute and where should it be added?

**Review by dstaley — APPROVED** [review]
Thank you!

**Pull request merged**